components/modal/modal.css

```diff
@@ -0,0 +1,69 @@
+.modal {
+  background-color: white;
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Can center a div using flex as well if using flex overall. This is fine as well.

Reply:
Yes that could be done, but since i had to use position:fixed for modal i thought might as well use it to centre the modal.